Update installation and ROCPD documentation (#1300)
ROCm/rocm-systems: projects/rocprofiler-systems/CHANGELOG.md

- ``ROCPROFSYS_ROCM_GROUP_BY_QUEUE`` configuration setting to allow grouping of events by hardware queue, instead of the default grouping.

> - Support for rocPD database output with the `ROCPROFSYS_USE_ROCPD` configuration setting.

Reviewer: ```suggestion
- Support for `rocpd` database output with the `ROCPROFSYS_USE_ROCPD` configuration setting.
```

> - Support for profiling Pytorch workloads using the rocPD output database.

Reviewer: ```suggestion
- Support for profiling PyTorch workloads using the `rocpd` output database.
```

- Support for tracing OpenMP API in Fortran applications.
- An error warning that is triggered if the profiler application fails due to SELinux enforcement being enabled. The warning includes steps to disable SELinux enforcement.

## Changed

- Updated the grouping of "kernel dispatch" and "memory copy" events in Perfetto traces. They are now grouped together by HIP Stream rather than separately and by hardware queue.

> - Updated PAPI module to v7.2.0b2

Reviewer: ```suggestion
- Updated PAPI module to v7.2.0b2.
```

> - Using ROCprofiler-SDK for tracing OMPT API calls.

Reviewer: ```suggestion
- ROCprofiler-SDK is now used for tracing OMPT API calls.
```
